tests/unit/Core/Checkout/Document/Renderer/OrderDocumentCriteriaFactoryTest.php

```diff
@@ -16,6 +17,7 @@
 #[CoversClass(OrderDocumentCriteriaFactory::class)]
 class OrderDocumentCriteriaFactoryTest extends TestCase
 {
+    #[DisabledFeatures(['v6.8.0.0'])]
```

Comment:
Do you expect this to be removed with 6.8? or just a few lines in the method?
I think, we should have one test with FF disabled, that is deprecated and one that is for the future.